importer_client/python/tools/timesketch_importer.py

```diff
@@ -143,10 +143,30 @@ def upload_file(
         timeline = streamer.timeline
         task_id = streamer.celery_task_id
 
+        analyzer_names = config_dict.get("analyzer_names")
+        timeline_ids = config_dict.get("timeline_ids")
```

Comment:
`analyzer_names` & `timeline_ids` are None here, since they are not set in the `config_dict` that is passed to this function.